diff --git a/dist/redis-pubsub.d.ts b/dist/redis-pubsub.d.ts
index 4bdc1abe0102c8dd505adc6cac10edb9cda3a9c0..ce7c74ba3b421f5ee8ec8edc402f87fd8474698e 100644
--- a/dist/redis-pubsub.d.ts
+++ b/dist/redis-pubsub.d.ts
@@ -24,7 +24,7 @@ export declare class RedisPubSub implements PubSubEngine {
     publish<T>(trigger: string, payload: T): Promise<void>;
     subscribe<T = any>(trigger: string, onMessage: OnMessage<T>, options?: unknown): Promise<number>;
     unsubscribe(subId: number): void;
-    asyncIterator<T>(triggers: string | string[], options?: unknown): AsyncIterator<T>;
+    asyncIterator<T>(triggers: string | string[], options?: unknown): AsyncIterable<T>;
     getSubscriber(): RedisClient;
     getPublisher(): RedisClient;
     close(): Promise<'OK'[]>;
